<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>storage</class>
 <widget class="QWidget" name="storage">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>400</height>
   </size>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::CustomContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>新增订单</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../res/resources.qrc">
    <normaloff>:/qt 图标/新增.jpg</normaloff>:/qt 图标/新增.jpg</iconset>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">
font: 75 12pt &quot;楷体&quot;;
background-color: rgb(144, 144, 144);
color:rgb(0, 0, 0)
</string>
  </property>
  <property name="inputMethodHints">
   <set>Qt::ImhNone</set>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="6" column="1" colspan="2">
    <widget class="QLineEdit" name="lineEdit_6">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <spacer name="verticalSpacer_4">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="0">
    <spacer name="verticalSpacer_6">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="6" column="3">
    <widget class="QLabel" name="label_11">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>单   位</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_2">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>订 单 号</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="label_12">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>生产日期    </string>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="10" column="0">
    <widget class="QLabel" name="label_6">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>产品数量</string>
     </property>
    </widget>
   </item>
   <item row="10" column="1" colspan="2">
    <widget class="QLineEdit" name="lineEdit_5">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
    </widget>
   </item>
   <item row="8" column="4">
    <widget class="QLineEdit" name="lineEdit_10">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
    </widget>
   </item>
   <item row="11" column="0">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="0">
    <spacer name="verticalSpacer_5">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0">
    <spacer name="verticalSpacer_7">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_4">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>产品编号</string>
     </property>
    </widget>
   </item>
   <item row="8" column="1" colspan="2">
    <widget class="QLineEdit" name="lineEdit_9">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
    </widget>
   </item>
   <item row="6" column="4">
    <widget class="QLineEdit" name="lineEdit_7">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
    </widget>
   </item>
   <item row="8" column="3">
    <widget class="QLabel" name="label_13">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>生成厂商</string>
     </property>
    </widget>
   </item>
   <item row="10" column="3">
    <widget class="QLabel" name="label_7">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>仓库编号</string>
     </property>
    </widget>
   </item>
   <item row="10" column="4">
    <widget class="QComboBox" name="comboBox">
     <property name="minimumSize">
      <size>
       <width>140</width>
       <height>0</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="4" column="1" colspan="2">
    <widget class="QLineEdit" name="lineEdit_3">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="2">
    <widget class="QLineEdit" name="lineEdit_1">
     <property name="cursor">
      <cursorShape>ForbiddenCursor</cursorShape>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="inputMethodHints">
      <set>Qt::ImhHiddenText</set>
     </property>
     <property name="inputMask">
      <string/>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Normal</enum>
     </property>
    </widget>
   </item>
   <item row="4" column="4">
    <widget class="QLineEdit" name="lineEdit_4">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
    </widget>
   </item>
   <item row="2" column="4">
    <widget class="QLineEdit" name="lineEdit_2">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_8">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>产品类别</string>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QLabel" name="label_3">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>产品名称</string>
     </property>
    </widget>
   </item>
   <item row="4" column="3">
    <widget class="QLabel" name="label_5">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>标签编号</string>
     </property>
    </widget>
   </item>
   <item row="12" column="0">
    <widget class="QLabel" name="label_10">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>管 理 员</string>
     </property>
    </widget>
   </item>
   <item row="12" column="4">
    <widget class="QLineEdit" name="lineEdit_8">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::NoEcho</enum>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="placeholderText">
      <string>NOW DATE</string>
     </property>
    </widget>
   </item>
   <item row="14" column="0" colspan="5">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>25</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
font: 75 11pt &quot;微软雅黑&quot;;
color:rgb(255, 255, 255);
background-color: rgb(83, 83, 83);
border-radius: 8px;
border: 3px groove rgb(98, 98, 98);
border-style: outset;
}
QPushButton:pressed{
border-style: inset;
}
QPushButton:hover{
background-color:rgb(0, 0, 0);
}

</string>
       </property>
       <property name="text">
        <string>确定</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_2">
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>25</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
font: 75 11pt &quot;微软雅黑&quot;;
color:rgb(255, 255, 255);
background-color: rgb(83, 83, 83);
border-radius: 8px;
border: 3px groove rgb(98, 98, 98);
border-style: outset;
}
QPushButton:pressed{
border-style: inset;
}
QPushButton:hover{
background-color:rgb(0, 0, 0);
}

</string>
       </property>
       <property name="text">
        <string>重置</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_3">
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>25</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
font: 75 11pt &quot;微软雅黑&quot;;
color:rgb(255, 255, 255);
background-color: rgb(83, 83, 83);
border-radius: 8px;
border: 3px groove rgb(98, 98, 98);
border-style: outset;
}
QPushButton:pressed{
border-style: inset;
}
QPushButton:hover{
background-color:rgb(0, 0, 0);
}

</string>
       </property>
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="12" column="1" colspan="2">
    <widget class="QComboBox" name="comboBox_2">
     <property name="minimumSize">
      <size>
       <width>145</width>
       <height>0</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="15" column="0">
    <spacer name="verticalSpacer_8">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="13" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="12" column="3">
    <widget class="QLabel" name="label_9">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>入库时间</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../res/resources.qrc"/>
 </resources>
 <connections/>
</ui>
